confirmation/models.py

```diff
@@ -73,7 +73,10 @@ def get_object_from_key(
 
 
 def create_confirmation_link(
-    obj: ContentType, confirmation_type: int, url_args: Mapping[str, str] = {}
+    obj: ContentType,
+    confirmation_type: int,
+    validity_in_days: Optional[int] = settings.INVITATION_LINK_VALIDITY_DAYS,
```

Comment:
why is this the defaut here, rather than `CONFIRMATION_LINK_DEFAULT_VALIDITY_DAYS` which we use as the default elsewhere?

Reply:
ohh i see, it's because this argument is for specifying the validity value for invitations specifically, makes sense

Reply:
Though I think the right approach to this function is to just accept the argument and apply it to any Confirmation created, if specificied - and if it isn't specified, use the default for the `confirmation_type`